rfc8826.xml

```diff
@@ -184,36 +218,75 @@ the title) for use on https://www.rfc-editor.org/search. -->
       <section anchor="sec.same-origin" numbered="true" toc="default">
         <name>Same-Origin Policy</name>
         <t>
-          Many other resources are accessible but isolated. For instance,
-          while scripts are allowed to make HTTP requests via the XMLHttpRequest() API (see <xref target="XmlHttpRequest" format="default"/>)
-          those requests are not allowed to be made to any server, but rather solely
-          to the same ORIGIN from whence the script came <xref target="RFC6454" format="default"/>
-          (although CORS <xref target="CORS" format="default"/> and WebSockets
+          Many other resources are accessible but isolated.
+          For instance,
+          while scripts are allowed to make HTTP requests via the fetch() API (see <xref target="fetch" format="default"/>)
+          when requests are made to a server other than from the
+          the same <strong>origin</strong> from whence the script came <xref target="RFC6454" format="default"/>
+          they not able to read the responses. Cross-Origin Resource Sharing (CORS) <xref target="CORS" format="default"/> and WebSockets
```

Comment:
You probably still need to update the CORS reference here to point to Fetch instead.